### ISSUE TITLE
Look into cache_choices has been deprecated warning.

### ISSUE THREAD

**Issue by theskumar** [Spoken-tutorial/spoken-website#70]
We are getting a lot of warning related to `cache_choices`, it would be nice to go with any alternatives to it as suggested by Django official doc.

```
spoken-website/events/forms.py:400: RemovedInDjango19Warning: cache_choices has been deprecated and will be removed in Django 1.9.
  language_prefered = forms.ModelChoiceField(cache_choices=True, widget = forms.Select(attrs = {}), queryset = Language.objects.order_by('name'), empty_label = "--- None ---", error_messages = {'required':'Language field required.'})
```

**Comment by harshitcodes** [comment]
@theskumar would [this](https://code.djangoproject.com/ticket/22841) be a good alternative to remove deprecated warnings related to `cache_choices`  which is proposed as an alternative in [https://code.djangoproject.com/ticket/22841](https://code.djangoproject.com/ticket/22841) ??

**Comment by theskumar** [comment]
Yes, that ticket has pretty good explanation on why the `cache_choices` is being removed. I'm not sure how useful they are currently, but if it doesn't help a lot it would good idea to get rid of them.
